extension/toggle-diff-changes.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+window.toggleDiffChanges = (() => {
+	const init = () => {
+		if ($('.js-collapse-all').length > 0) {
+			return;
+		}
+
+		const html = `<button type="button" class="btn btn-sm js-collapse-all">Collapse All</button>`;
```

Comment:
@DrewML Indeed, we just need someone to do it: https://github.com/sindresorhus/eslint-plugin-unicorn/issues/71